### PR TITLE
16729: 1990 Content update

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
As a developer, I need to update the 22-1990 form so that it is consistent with DEPO recommendations and other Education Services forms.

[Originating Issue](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/16729)

## Testing done
Manual Testing Passes Locally

## Screenshots
Screen Shot 2020-12-08 at 12 21 42 PM
Screen Shot 2020-12-08 at 12 25 23 PM
Screen Shot 2020-12-08 at 12 25 29 PM
Screen Shot 2020-12-08 at 12 25 35 PM
Screen Shot 2020-12-08 at 12 25 43 PM
Screen Shot 2020-12-08 at 12 25 50 PM
Screen Shot 2020-12-08 at 12 25 55 PM
Screen Shot 2020-12-08 at 12 26 01 PM

## Acceptance criteria
- [x] The following chapter titles are sentence case:
a. Applicant information
b. Benefits eligibility
c. Service history (formerly Military History)
d. Education history
e. Employment history
f. School selection
g. Personal information

- [x] On the submission confirmation page:
a.
Claim received
is replaced with
We’ve received your application

b.
Education Claim (Form 22-1990)
is replaced with
Education benefit application (Form 22-1990)

c. The content in the "What happens after I apply?" section is:
We usually decide on applications within 30 days.
You’ll get a Certificate of Eligibility (COE) or decision letter in the mail. If we’ve approved your application, you can bring the COE to the VA certifying official at your school.
Learn more about what happens after you apply

d. The padding above "Education Benefit Claim (Form 22-1990)" on the confirmation submission screen is reduced. (Ex. the 10203 has the class vads-u-margin-top--0; per UX 23px is the appropriate amount of space).

e. The "Need help?" section style matches DEPO patterns (see SA2).

f. The confirmation page layout and styles match that as shown in SA#3

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/edu-benefits/1990/containers/ConfirmationPage.jsx [hunk 1]
```diff
@@ -161,11 +165,20 @@ class ConfirmationPage extends React.Component {
...
           </h4>
           <p className="confirmation-guidance-message">
+            We usually decide on applications within 30 days.
+            <br />
+            <br />
+            You’ll get a Certificate of Eligibility (COE) or decision letter in
+            the mail. If we’ve approved your application, you can bring the COE
+            to the VA certifying official at your school.
+            <br />
+            <br />
```
Comment (shawkey-daniel) [inline src/applications/edu-benefits/1990/containers/ConfirmationPage.jsx:175]: Is there a better way to style the whitespace here?

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: src/applications/edu-benefits/1990/containers/ConfirmationPage.jsx (hunk 1 above)
Comment (elder-joseph) [reply src/applications/edu-benefits/1990/containers/ConfirmationPage.jsx:175]: I can modify that a bit. The double `<br />` is certainly not ideal.

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]
lgtm

**Review by billfienberg — APPROVED** [review]

**Pull request merged**